src/extensions/lens-main-extension.ts

```diff
@@ -7,7 +7,7 @@ export class LensMainExtension extends LensExtension {
     //
   }
 
-  registerStatusBarIcon(registry: StatusBarRegistry) {
+  registerStatusBarItem(registry: StatusBarRegistry) {
```

Comment:
This was added by accident, there is no "status bar" in main process :) So plz remove.

Reply:
Yes, this should be removed.